### PR TITLE
feat: update German translation

### PR DESCRIPTION
**Pull request by j9t**
Signed-off-by: Jens Oliver Meiert <jens@meiert.com>

### CODE REVIEW

File: questions/qa-http-and-lang.de.html [hunk 1]
```diff
@@ -47,116 +47,122 @@
...
-<ul id="full-links">
+  <h2 id="endlinks" class="notoc"><a href="#endlinks">Literaturhinweise</a></h2>
+  <aside class="section" id="survey"></aside>
+  <script>
+    document.getElementById('survey').innerHTML = g.survey
+  </script>
+  <ul id="full-links">
     <li>
-      <p>Erste Schritte: <a href="/International/getting-started/language"><cite>Sprachangaben im Web</cite></a></p>
+      <p>Erste Schritte: <a href="https://www.w3.org/International/getting-started/language"><cite>Sprachangaben im Web
```
Comment (xfq) [inline questions/qa-http-and-lang.de.html]: Although I don't know why, https://www.w3.org/International/i18n-drafts/pages/translation#checklist says "Links to files under https://www.w3.org/International start with /International".

Comment (j9t) [reply questions/qa-http-and-lang.de.html]: Interesting. This is coming straight from the original version (FYI @r12a). I trimmed this and other links as the translation checklist is explicit about this point. Thanks!

**Comment by r12a** [comment]
@j9t this looks good. Thanks.

I just added some text to the beginning of the document as discussed in https://github.com/w3c/i18n-drafts/issues/338, so if you could update your PR to include that we should be good to go.
See https://github.com/w3c/i18n-drafts/pull/342

Thanks.

**Comment by j9t** [comment]
Synced this PR with your changes in #342. Still have to translate the new paragraph—couldn’t make this a draft PR but will update here once done.

**Comment by j9t** [comment]
@r12a, @xfq, the translation was amended and is ready for review again.

**Comment by r12a** [comment]
Thanks @j9t.  I'm happy to merge this now, unless you envisage other steps before that?

**Comment by j9t** [comment]
Thanks! No, good to go from my end. I’ll review the end result again, too—if I notice anything I’ll file another PR.

**Comment by r12a** [comment]
Ok, now published at https://www.w3.org/International/questions/qa-http-and-lang.de and translation warning removed.  Thanks again for the translation.

**Pull request merged**